Cleaned up CSS by factoring out commonly used components

dfangshuo opened the pull request:
## Cleaned up CSS by factoring out commonly used components 

Main idea is to factor out common styles into a `main.css` file e.g. pink rounded buttons, square gray buttons etc.

commonly used styles in `main.css`
cc-ing @ethanlee16 on this one cause im not 100% sure what the best practice is here

### Related PRs


### Migrations

### Tests Performed, Edge Cases

### Screenshots

CC: @aivantg @dfangshuo

ethanlee16 commented on src/styles/main.css:
```diff
@@ -0,0 +1,63 @@
+}
+
+.pp-pink-rounded-button {
+  -webkit-transition-duration: 0.4s; /* Safari */
```
You shouldn't need any vendor-specific prefixes on your properties, since [CRA ships with Autoprefixer](https://create-react-app.dev/docs/post-processing-css/).

ethanlee16 commented on src/screens/onboarding/Onboarding.js:
```diff
@@ -85,7 +85,7 @@ class Onboarding extends React.Component {
         billingZipcode: ''
       },
-      step: 1
+      step: 2
```
Is this change intentional? It doesn't seem related to this PR, or I can't find anything on why this is included.

dfangshuo commented:
Cleaned up CSS files (including deleting some unused files) and switched things over to use kebab case consistently. Might leave switching to some class name convention in a future PR because i trie doing it and realized the changes would be too massive
